src/cml.js

```diff
@@ -1,49 +1,58 @@
 const { execSync } = require('child_process');
 const git_url_parse = require('git-url-parse');
 const strip_auth = require('strip-url-auth');
-const git = require('simple-git/promise')('./');
 const globby = require('globby');
+const git = require('simple-git/promise')('./');
 
 const Gitlab = require('./drivers/gitlab');
 const Github = require('./drivers/github');
 const BitBucketCloud = require('./drivers/bitbucket_cloud');
 const { upload, exec, watermark_uri } = require('./utils');
 
+const {
+  GITHUB_REPOSITORY,
+  CI_PROJECT_URL,
+  BITBUCKET_REPO_UUID,
+  CI
+} = process.env;
+
+const GIT_USER_NAME = 'iterative-olivaw';
+const GIT_USER_EMAIL = 'olivaw@iterative.ai';
```

Comment:
```suggestion
const GIT_USER_NAME = 'Olivaw[bot]';
const GIT_USER_EMAIL = '64868532+iterative-olivaw@users.noreply.github.com';
```

Reply:
Why? In Bitbucket and Gitlab those users exists under that email?

Reply:
ah I see, sure OK. At least the (display) name should be changed to look like a bot though:


```suggestion
const GIT_USER_NAME = 'Olivaw[bot]';
const GIT_USER_EMAIL = 'olivaw@iterative.ai';
```

Reply:
I took the one in our notion. Is Olivaw[bot] fine or do we have to setup it in place?

Reply:
Git commits can use any username we want, it doesn't have anything to do with GH/GL/BB username. Notion says use `Olivaw[bot]` for git commits...

Only the email needs to match for GH/GL/BB to pick up the user.

Reply:
> Only the email needs to match for GH/GL/BB to pick up the user.

I think I tried sometime ago and it did not work and made sense since anyone could relate a slang word with a real account. Let me confirm

Reply:
https://github.com/amoffat/masquerade/commit/9b0562595cc479ac8696110cb0a2d33f8f2b7d29

Reply:
In BB does not work. Not sure in Gitlab. Lets try, as i say I remember to test this and in one or the other the user if not totally consistent was not showing all the details (gravatar, etc...)

Reply:
Seems to be working. We have to update the profile Gravatars